meta/src/parser.rs

```diff
@@ -1347,7 +1419,11 @@ mod tests {
                             ))))
                         ))
                     ))))))
-                )
+                ),
+                comments: vec![
+                    "This is line comment".to_string(),
+                    "This is rule".to_string(),
+                ],
```

Comment:
> so having an assertion that [" hello", " world"] was correctly extracted out (just like in the ast test that asserts that the line/rule comments were extracted)

@tomtau Do you means this test?

In this test case, it covered for extract `/// This is rule` line comments into `["This is line comment", "This is rule"]`.

Reply:
yes